lib/index.js

```diff
@@ -51,9 +51,9 @@ const Rooftop = {
       contentType: (name) => {
         return {
           get: (opts = {}) => {
```

Comment:
When opts isn't defined, you'll want the default value to have a params key with a blank object, like this:
`get: (opts = { params: {} }) => {`

Otherwise the call to `Object.keys(opts.params)` on line 54 will generate an error.